packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -93,6 +167,39 @@ export class RichTextEditor extends FoundationElement {
     public override disconnectedCallback(): void {
         super.disconnectedCallback();
         this.unbindEditorTransactionEvent();
+        this.unbindEditorUpdateEvent();
+        this.resizeObserver?.disconnect();
+    }
+
+    /**
+     * @internal
+     */
+    public disabledChanged(): void {
+        if (this.tiptapEditor) {
+            this.tiptapEditor.setEditable(!this.disabled);
+            this.editor.setAttribute(
+                'aria-disabled',
+                this.disabled ? 'true' : 'false'
+            );
+        }
+    }
+
+    /**
+     * Update the placeholder text and view of the editor.
+     * @internal
+     */
+    public placeholderChanged(): void {
+        if (this.tiptapEditor) {
+            const placeholderExtension = this.tiptapEditor.extensionManager.extensions.filter(
+                extension => {
+                    return extension.name === 'placeholder';
+                }
+            )[0] as Extension<PlaceholderOptions>;
```

Comment:
Consider moving this into a separate function `getTipTapExtension(extensionName: string)`